x-pack/plugins/synthetics/server/synthetics_service/service_api_client.ts

```diff
@@ -139,10 +159,28 @@ export class ServiceAPIClient {
     method: 'POST' | 'PUT' | 'DELETE',
     { monitors: allMonitors, output, runOnce, isEdit }: ServiceData
   ) {
+    if (!this.server.coreStart) {
```

Comment:
@shahzad31 Should I add a warning log for this?

Reply:
it's code debt actually. There is no need to add a log here.

Reply:
This was removed entirely.